<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_modifyAdherent</class>
 <widget class="QWidget" name="Add_modifyAdherent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>361</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="verticalSpacing">
     <number>50</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>NOM   :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nomEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>PRENOM :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="PrenomEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Sexe:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>TEL        :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="TelEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>     ADRESSE    :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="AddressseEdit"/>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_h">
        <property name="text">
         <string>Homme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_f">
        <property name="text">
         <string>Femme</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>3</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="valider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enregister</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ADHERENT</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
